graphql.go

```diff
@@ -230,6 +231,13 @@ func WithBeforeRetryHandler(beforeRetryHandler func(*http.Request, *http.Respons
 	}
 }
 
+// WithDefaultHeader provides a default set of header values
+func WithDefaultHeader(defaultHeader map[string]string) ClientOption {
```

Comment:
if this allows setting multiple header values, it should be called `WithDefaultHeaders`(plural)

Reply:
ok!